src/components/Email/Email.tsx

```diff
@@ -21,8 +22,14 @@ const Email:React.FC<IProps> = ({component, openComponent}) => {
         openComponent(true)
     }
 
-    const submitBtn = (e:any) => {
+    const submitBtn = () => {
         openComponent(!component)
+        openComponentShipping(true)
```

Comment:
Change naming a bit, to have a clear picture of what is opening and what is closing

Reply:
I changed it to OpenComponentEmail because it's referring to email component.